packages/terra-table/src/SelectableTableRows.jsx

```diff
@@ -25,12 +25,27 @@ const propTypes = {
    */
   // eslint-disable-next-line react/forbid-prop-types
   selectedIndexes: PropTypes.array,
+  liveRegion: PropTypes.oneOfType([
```

Comment:
Can we add some doc for these props?

Reply:
https://github.com/cerner/terra-core/pull/2331/commits/3fb9b80ad6657aec97c4c26e072c7f421c0e62fc